### PR TITLE
Update user guide stuff

### LINKED ISSUE #98: Update AddWindow and EditWindow icons to be more representative

### PR DESCRIPTION
**Pull request by bakano98**
In this PR, a few things are done.

For the user guide:
- Fix JAR file link
- Fix editWindow.png
- Group relevant commands together (e.g. all add should be together in 1 section)
- Have a small section below quick jump to explain that sections are the list of available commands for users
- Rename Quick Jump to Quick Jump to Features and move to the start of the document
- Edit add's example
- Copy section has an out of place table
- Change e.g to e.g. (note the extra dot)
- Change HelpWindow's image

I have also updated the icons:
1. AddWindow
2. EditWindow

Closes #98 and #99

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,21 +3,43 @@ layout: page
 title: User Guide
 ---
```
Comment (lawwm) [inline docs/UserGuide.md:4]: I know these wasn't modified by your commit, but do you know what this table is for?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -3,21 +3,43 @@ layout: page
...
+ModuleMateFinder (MMF) is a **desktop app for managing contacts in your educational organization, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, MMF can get your contact management tasks done faster than traditional GUI apps.
 
 - [Quick Start](#quick-start)
 - [Features](#features)
-- [Quick Jump](#quick-jump)
 - [FAQ](#faq)
 - [Command Summary](#command-summary)
 
 --------------------------------------------------------------------------------------------------------------------
+## Quick Jump to Specific Features
```
Comment (lawwm) [inline docs/UserGuide.md:14]: I think "Shortcut" might be better than "Quick Jump", especially since already have "Quick Start" nearby.

**Review by lawwm — COMMENTED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (bakano98) [reply docs/UserGuide.md:4]: Not sure, but I think it's being used for GitHub pages.
It corresponds to this:
image

File: docs/UserGuide.md (hunk 2 above)
Comment (bakano98) [reply docs/UserGuide.md:14]: Good point. Will change.

**Review by Btaykb — COMMENTED** [review]
LGTM

**Pull request merged**